Makefile.am

```diff
@@ -202,7 +201,7 @@ precompute_ecmult_gen_LDADD = $(COMMON_LIB)
 # otherwise make's decision whether to rebuild them (even in the first
 # build by a normal user) depends on mtimes, and thus is very fragile.
 # This means that rebuilds of the prebuilt files always need to be
-# forced by deleting them, e.g., by invoking `make clean-precomp`.
+# forced by deleting them, e.g., using `make maintainer-clean`.
```

Comment:
Considering that `make maintainer-clean` [removes](https://github.com/bitcoin-core/secp256k1/pull/1277#issuecomment-1514815590) `Makefile`, could this comment be confusing if someone follows it blindly?

Reply:
I just removed that part.